### PR TITLE
Implementing trigonometric functions

### PR DESCRIPTION
**Pull request by FlorentinD**
* functions that are currently not supported in spark are evaluated based on existing functions (*haversin()* and *cot()*).

### CODE REVIEW

File: spark-cypher-testing/src/test/scala/org/opencypher/spark/impl/acceptance/FunctionsBehaviour.scala [hunk 1]
```diff
@@ -36,6 +36,343 @@ import org.scalatest.DoNotDiscover
 @DoNotDiscover
 class FunctionsBehaviour extends CAPSTestSuite with DefaultGraphInit {
 
+  describe("Acos"){
+    it("on int value") {
+      val result = caps.cypher("Return acos(1) as res")
```
[the remark this answers is not in the document]
Comment (FlorentinD) [reply spark-cypher-testing/src/test/scala/org/opencypher/spark/impl/acceptance/FunctionsBehaviour.scala]: sounds good.
edited it in the whole file :)

**Comment by FlorentinD** [comment]
Unfortunatly I cannot see why the check is failing as I have no login for TeamCity.
Locally the edited tests in the last commit are all running through.

**Comment by s1ck** [comment]
It's failing because of:
```
[10:25:12][Step 3/5] [INFO] --- license-maven-plugin:3.0:check (check-license-headers) @ spark-cypher-testing ---
[10:25:12][Step 3/5] [INFO] Checking licenses...
[10:25:12][Step 3/5] [WARNING] Missing header in: /mnt/var/opt/teamcity-agent/work/2118b2d0507e3be3/spark-cypher-testing/src/test/scala/org/opencypher/spark/impl/acceptance/FunctionsBehaviour.scala
[10:25:12][Step 3/5] Failed to execute goal com.mycila:license-maven-plugin:3.0:check (check-license-headers) on project spark-cypher-testing: Some files do not have the expected license header
```
Just run:
```
mvn license:format
```
in the project root and commit.

File: spark-cypher-testing/src/test/scala/org/opencypher/spark/impl/acceptance/FunctionsBehaviour.scala [hunk 2]
```diff
@@ -8,7 +8,7 @@
  *     http://www.apache.org/licenses/LICENSE-2.0
  *
  * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "as IS" BASIS,
+ * distributed under the License is distributed on an "AS IS" BASIS,
```
Comment (s1ck) [inline spark-cypher-testing/src/test/scala/org/opencypher/spark/impl/acceptance/FunctionsBehaviour.scala]: :)

Comment (FlorentinD) [reply spark-cypher-testing/src/test/scala/org/opencypher/spark/impl/acceptance/FunctionsBehaviour.scala]: fixed in my latest commit; replacing words is harder than I thought :D

**Comment by s1ck** [comment]
Looks nice. As discussed offline, @FlorentinD will reword the commit messages to align with our scheme and then we can merge. Thanks for the contribution!

**Pull request merged**